data-service/v1/dataservice.proto

```diff
@@ -0,0 +1,39 @@
+//
+// DISCLAIMER
+//
+// Copyright 2023 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+syntax = "proto3";
+
+package arangodb.cloud.integration.bucketservice.v1;
+
+option go_package = "github.com/arangodb-managed/integration-apis/bucket-service/v1";
+
+import "common/v1/common.proto";
+import "common/v1/icommon.proto";
+import "data/v1/data.proto";
+
+import "github.com/golang/protobuf/ptypes/timestamp/timestamp.proto";
+
+// DataService is the API used to manage deployments.
+service DataService {
```

Comment:
IMO it make it consistent it should be DataServiceService, on Data we have a DataService already.

Reply:
Okay! But `DataServiceService` sounds a bit funny :)

Reply:
Sorry, not sure I understand correctly. But we already have a `BucketService`: https://github.com/arangodb-managed/integration-apis/blob/master/bucket-service/v1/bucketservice.proto#L33

Reply:
Sorry same confusing with internal-api here, this is correct!